dev/plugins/hu.elte.txtuml.export.javascript/src/hu/elte/txtuml/export/javascript/resources/res/visualize.html

```diff
@@ -80,8 +80,12 @@
 					<input type="number" name="debugPort" id="debug-port-input">
 				</div>
 				<div style="flex: 0 0">
+					<br>
 					<label class="debug-label" for="animationDelay">State machine's slow-down</label>
-					<input type="range" name="animationDelay" value="1" min="1" max="10" id="animation-delay-input">
+					<input type="range" name="animationDelay" value="1" min="1" max="10" id="animation-delay-slider">
```

Comment:
I have some additional design concerns:

- we should display the unit, secons, in my opinion
- the input box should be streched or moved to center

Also correct me if I remember wrong, but we agreed on making the input box non-editable, no?